### DISCUSSION TITLE
Welcome to Positron! 🎉

### DISCUSSION THREAD

**Discussion by juliasilge** [posit-dev/positron#3363] [Announcements]
We are so excited to be sharing our work on Positron with you, one of the first people to be using this new IDE. 🙌

Check out:

- [our website](https://positron.posit.co/) for information on what you should do before installing Positron, troubleshooting Positron, and more 📑
- the [Releases](https://github.com/posit-dev/positron/releases) page, where you can obtain an installer for a pre-release build 🚀

We invite you to use Discussions here to [ask questions and share feedback](https://github.com/posit-dev/positron/wiki/Feedback-and-Issues).

**Comment by bbrewington** [comment]
@juliasilge could you please update the "our wiki" link to point to https://positron.posit.co/ ?

**Reply by juliasilge** [reply]
Thanks for the heads up! ✅

**Comment by cjimenezgallardo** [comment]
Hi,,
I'm using for the first time, (I use rstudio), and I built a function, which shows a summary of statistics and two graphs and display them in plot through a print(), but it only shows the second one (bp)

![image](https://github.com/user-attachments/assets/28ba9876-8b23-4955-bdf2-e789dee9fbb7)


thanks for the effort in making this IDE

**Reply by jmcphers** [reply]
I think you're seeing another instance of https://github.com/posit-dev/positron/issues/727 -- follow that issue for updates. Thanks for reporting!
